pages/_app.tsx

```diff
@@ -25,6 +27,12 @@ const wagmiConfig = defaultWagmiConfig({
 
 createWeb3Modal({ wagmiConfig, projectId, chains });
 
+initWeb3InboxClient({
+  projectId,
+  allApps: true,
+  domain: "gm.walletconnect.com",
```

Comment:
```suggestion
  ...(process.env.NODE_ENV != "production" ? {
    allApps: true,
    domain: "gm.walletconnect.com",
  } : {})
```

Reply:
Ah yes forgot that while debugging

Reply:
Thanks; we should probably put this example in the docs too